rasa_nlu/cross_eval.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from rasa_nlu.converters import load_data
+from rasa_nlu.config import RasaNLUConfig
+from rasa_nlu.model import Trainer, TrainingData, Interpreter
+
+from sklearn.model_selection import StratifiedKFold
```

Comment:
Needs to be scoped (e.g. put it in the first lines of the functions that need it - it is an optional dependency).